napari/layers/_markers_layer/view/properties.py

```diff
@@ -93,7 +93,10 @@ def changeSymbol(self, text):
         self.layer.symbol = text
 
     def changeSize(self, value):
-        self.layer.size = value
+        avg = np.asarray(self.layer.size).mean()
+        if avg == 0:
+            avg = 1
+        self.layer.size = self.layer.size/avg*value
```

Comment:
pep8 and maybe some parentheses might be nice

Reply:
fixed